tests/test_ingest.py

```diff
@@ -28,6 +28,7 @@
 
 from lsst.daf.butler import Butler, DataCoordinate
 from lsst.obs.base.ingest_tests import IngestTestBase
+import lsst.afw.cameraGeom.testUtils
```

Comment:
This seems wrong to me. This file does not call `assertDetectorsEqual`. This line should be added instead to `obs_base/python/lsst/obs/base/ingest_tests.py`.